### PR TITLE
Use Snapshot X quest

### PR DESCRIPTION
**Pull request by zzuziak**
This PR adds a guide for Linea members to use [Snapshot X](https://docs.snapshotx.xyz)

### CODE REVIEW

File: docs/use-linea/explore/use-snapshotx.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* **Space** - You can think of a space as an organization's account on Snapshot which can be viewed by anyone visiting the platform. It serves as a hub for all proposals related to the organization and a source of information for the users.
+* **Proposal** - Proposal is the key element of the voting system. It presents a change suggestion related to a specific organization and enables eligible users to cast their vote. 
+* **Voting Strategies** - Voting strategies are the contracts used to determine the voting power (VP) of users.
+
+
+# Voting
+
+## Who can vote on proposals? <a href="#who-can-vote-on-proposals" id="who-can-vote-on-proposals"></a>
+
+Each space specifies their [Voting Strategies](https://docs.snapshotx.xyz/protocol-sx-evm/voting-strategies.md) in its settings. You can see the custom setup by opening the space settings. This setup can define if you are eligible to take part in the voting and what is your Voting Power calculated at the timestamp of proposal creation.
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-snapshotx.md:23]: @zzuziak this is a dead link

Comment (emilyJLin95) [reply docs/use-linea/explore/use-snapshotx.md:23]: @zzuziak 
can you change, squash your commits, and force push?
lmk if you need help with the process

File: docs/use-linea/explore/use-snapshotx.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+In case of Linea at the time of the quest every address has 1 Voting Power.
+
+
+# Quest - Cast a vote <a href="#cast-a-vote" id="cast-a-vote"></a>
+
+{% hint style="info" %}
+On-chain voting on Snapshot X doesn't cost you anything nor does it affect your funds in any way.
+{% endhint %}
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-snapshotx.md:32]: this is not a real word

can you update `project-words.txt` to allowlist non-real words?
You can see the failures here:

Screenshot 2023-06-13 at 10 06 08 AM

And then run `npm run lint` afterwards to check?

lmk if you need help

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docs/use-linea/explore/use-snapshotx.md (hunk 1 above)
Comment (zzuziak) [reply docs/use-linea/explore/use-snapshotx.md:22]: Removed both links 👌  Couldn't squash though, tried to `rebase -i` but your project prompt went through each commit message individually instead 🤔  feel free to fetch and squash!

File: docs/use-linea/explore/use-snapshotx.md (hunk 2 above)
Comment (zzuziak) [reply docs/use-linea/explore/use-snapshotx.md]: Added 👌 
Just to make sure with the `{% hint` syntax - you are using Gitbook right? If so it should render it correctly.

File: docs/use-linea/explore/use-snapshotx.md (hunk 1 above)
Comment (emilyJLin95) [reply docs/use-linea/explore/use-snapshotx.md:22]: Did you push your changes? the dead link is still there
@zzuziak

**Comment by zzuziak** [comment]
Closing this one due to conflicts, new **[PR here](https://github.com/ConsenSys/doc.zk-evm/pull/131)**

**Pull request closed without merging**